demisto_sdk/commands/common/hook_validations/layout.py

```diff
@@ -275,6 +278,59 @@ def is_id_equals_name(self):
         """
         return super()._is_id_equals_name("layoutscontainer")
 
+    @error_codes("LO107")
+    def is_valid_mpv2_layout(self):
+        invalid_sections = [
+            "evidence",
+            "childInv",
+            "linkedIncidents",
+            "team",
+            "droppedIncidents",
+            "todoTasks",
+            "invTimeline",
+        ]
+        invalid_tabs = ["canvas", "evidenceBoard", "relatedIncidents"]
+        invalid_types_contained = []
+
+        marketplace_versions = get_item_marketplaces(
+            self.file_path, item_data=self.current_file
+        )
+        if MarketplaceVersions.MarketplaceV2.value not in marketplace_versions:
+            return True
```

Comment:
??

Reply:
this validation does not relevant for layouts not for XSIAM